addons/website/views/snippets/s_pricelist_boxed.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_pricelist_boxed" name="Pricelist Boxed">
+    <section class="s_pricelist_boxed pt72 pb72 o_colored_level parallax s_parallax_is_fixed s_parallax_no_overflow_hidden" data-scroll-background-ratio="1">
+        <span class="s_parallax_bg oe_img_bg o_bg_img_center" style="background-image: url('/web/image/website.s_pricelist_boxed_default_background');"/>
+        <div class="o_container_small">
```

Comment:
Rather than setting the column as a `col-lg-12` in a `o_container_small`, I'd switch to `container` and use `offset-lg-2 col-lg-9` instead, this way the card  is "resizable" by default.